nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-processors/pom.xml

```diff
@@ -56,6 +56,11 @@ language governing permissions and limitations under the License. -->
             <version>${es.version}</version>
         </dependency>
         <dependency>
+            <groupId>com.squareup.okhttp</groupId>
+            <artifactId>okhttp</artifactId>
+            <version>2.7.1</version>
```

Comment:
Turns out the API is very similar (mostly just switched from get/set to a Builder), so we're good to go :)